test/db/analysis/v850

```diff
@@ -5,72 +5,72 @@ aap
 afl
 EOF
 EXPECT=<<EOF
-0x00100018    1 8            fcn.00100018
```

Comment:
Have checked validity of these changes?

Reply:
I checked some but not all of them with IDA.

Reply:
Actually, one of the things I've found is that when `aaa` has been used to do a full analysis, the information on the functions is much more accurate.

Reply:
> @imbillow accurate compared to what?

Compare the output of `afl`, which contains the addr bbs size real_size for each function